pkgs/development/python-modules/pyTwitchAPI/default.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, aiohttp
+, python-dateutil
+, requests
+, websockets
+}:
+
+buildPythonPackage rec {
+  pname = "pyTwitchAPI";
```

Should follow [PEP-503](https://nixos.org/manual/nixpkgs/stable/#contributing).

```suggestion
  pname = "pytwitchapi";
```

---

Changes implemented